variables.tf

```diff
@@ -14,6 +14,12 @@ variable "sftp_users" {
   description = "List of SFTP usernames and public keys"
 }
 
+variable "sftp_restricted" {
```

Comment:
Let's call it restricted home

Reply:
`home_directory_type` would keep it consistent with the argument.

Reply:
I'm leaning towards `restricted_home` because `home_directory_type` on the `aws_transfer_user` resource doesn't entirely control whether or not the user will be restricted to their home directory.